presentation/matrix_view.py

```diff
@@ -48,16 +52,20 @@ def make_data(mesh: MeshResults, matrix: Matrix, tresholds: Thresholds) -> List[
     ]
 
 
-def make_colors(mesh: MeshResults, matrix: Matrix, tresholds: Thresholds) -> List[List]:
+def make_colors(mesh: MeshResults, metric: str, matrix: Matrix, tresholds: Thresholds) -> List[List]:
     colors = []
     for row in reversed(mesh.rows):
         colors_col = []
         for col in row.columns:
             deteriorated = tresholds.deteriorated(row.agent_id, col.agent_id)
             failed = tresholds.failed(row.agent_id, col.agent_id)
-            color = get_color(
-                matrix.cells[row.agent_alias][col.agent_alias].latency_microsec.value, deteriorated, failed
-            )
+            if metric == "latency":
```

Comment:
Please extract `get_metric_value()` function that takes `matrix.cells[row.agent_alias][col.agent_alias]`  and  `metric` and returns metric value.

Reply:
Done